src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -998,13 +1002,17 @@ def set_tag(self, key, value):
         )
         self.tags[key] = value
 
+    def get_device_type(self):
+        if self.tags.get('ceph.type'):
+            return self.tags.get('ceph.type')
+        return 'data'
+
```

Comment:
This seems wrong. Using this any LV (even non ceph related) are of type `data`?

This should also be written as `return self.tags.get('ceph.type', None)` and frankly this can be done by any caller without loss of convenience.